### PR TITLE
ENG-1353: FE - Change the widget config screen

### PR DESCRIPTION
**Pull request by gReis89**
This is a workaround to disable the widgets config screens. As we can have many different configs, accordingly to the widget needs. The best approach would create a `readOnly` state for each widget we support, but keeping this standard would be very difficult.

So what I did was:

- Removing the Cancel and Save buttons by using a generic class selector: `[class*=save]` which would cover all our existing cases. The Cancel as Save buttons are created by each widget config form, making it difficult to generic hide them.
- added a grey layer with opacity over the form, making it impossible for the users to trigger anything.
- Added warnings to inform users how to enable the form.
- Added a button to lead users to the widget default configs

### CODE REVIEW

**Comment by entando-jx** [comment]
[APPROVALNOTIFIER] This PR is **NOT APPROVED**

This pull-request has been approved by:
To fully approve this pull request, please assign additional approvers.
We suggest the following additional approver: **greis89**

If they are not already assigned, you can assign the PR to them by writing `/assign @greis89` in a comment when ready.

The full list of commands accepted by this bot can be found [here](https://go.k8s.io/bot-commands?repo=entando%2Fapp-builder).

The pull request process is described [here](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process)

<details open>
Needs approval from an approver in each of these files:

- **[OWNERS](https://github.com/entando/app-builder/blob/master/OWNERS)**

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

File: src/ui/widgets/config/WidgetConfigPage.js [hunk 1]
```diff
@@ -16,6 +16,23 @@ import { isMicrofrontendWidgetForm } from 'helpers/microfrontends';
 import WidgetConfigMicrofrontend from 'ui/widgets/config/WidgetConfigMicrofrontend';
 
 
+const msgs = defineMessages({
+  widgetConfigError: {
+    id: 'widget.page.config.error',
+    defaultMessage: 'NaUnable to load widget configurationme',
```
Comment (gidesan) [inline src/ui/widgets/config/WidgetConfigPage.js:22]:
```suggestion
    defaultMessage: 'Unable to load widget configuration',
```

**Review by gidesan — CHANGES_REQUESTED** [review]

**Comment by entando-jx** [comment]
@gReis89: The following test **failed**, say `/retest` to rerun them all:

Test name | Commit | Details | Rerun command
--- | --- | --- | ---
pr-build | 48bf731dcf51fd26bc6f3ca05fd409522199dbdb | [link]() | `/test this`



<details>

Instructions for interacting with me using PR comments are available [here](https://git.k8s.io/community/contributors/guide/pull-requests.md).  If you have questions or suggestions related to my behavior, please file an issue against the [kubernetes/test-infra](https://github.com/kubernetes/test-infra/issues/new?title=Prow%20issue:) repository. I understand the commands that are listed [here](https://go.k8s.io/bot-commands).
</details>

File: sass/pages/config/PageConfigPage.scss [hunk 2]
```diff
@@ -98,6 +98,30 @@
...
+  }
+
+  &__block-ui {
+    display: block;
+    position: absolute;
+    top: 0;
+    left: 0;
+    width: 100%;
+    height: 100%;
+    background-color: rgba(0,0,0,.1); // sass-lint:disable-line variable-for-property
```
Comment (gidesan) [inline sass/pages/config/PageConfigPage.scss:113]: why not follow the linter here? not a big deal to add a variable, I guess

File: src/ui/widgets/config/WidgetConfigPage.js [hunk 3]
```diff
@@ -1,21 +1,41 @@
...
 
+const msgs = defineMessages({
+  widgetConfigError: {
+    id: 'widget.page.config.error',
+    defaultMessage: 'Unable to load widget configuration',
+  },
+});
+
+function removeActionsButton(wrapper) {
+  const saveButton = wrapper.querySelectorAll('[class*=save]')[0];
```
Comment (gidesan) [inline src/ui/widgets/config/WidgetConfigPage.js:28]: why direct DOM manipulation here?

**Review by gidesan — CHANGES_REQUESTED** [review]

File: src/ui/widgets/config/WidgetConfigPage.js (hunk 3 above)
Comment (gidesan) [reply src/ui/widgets/config/WidgetConfigPage.js:28]: after a short conversation, it's caused by some technical debt we have to solve

**Review by gidesan — APPROVED** [review]

**Comment by entando-jx** [comment]
[APPROVALNOTIFIER] This PR is **APPROVED**

This pull-request has been approved by: *<a href="https://github.com/entando/app-builder/pull/864#pullrequestreview-508313388" title="Approved">gidesan</a>*

The full list of commands accepted by this bot can be found [here](https://go.k8s.io/bot-commands?repo=entando%2Fapp-builder).

The pull request process is described [here](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process)

<details >
Needs approval from an approver in each of these files:

- ~~[OWNERS](https://github.com/entando/app-builder/blob/master/OWNERS)~~ [gidesan]

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

**Pull request merged**